build-frontend/action.yml

```diff
@@ -2,71 +2,83 @@ name: Build Frontend
 description: Builds Storefront app docker image
 inputs:
   frontend:
-    description: 'Switch between Next.js and Nuxt'
+    description: "Switch between Next.js and Nuxt"
     required: false
-    default: 'next'
+    default: "next"
   project_name:
-    description: 'Project name'
+    description: "Project name"
     required: true
   docker_registry_url:
-    description: 'Docker registry url'
+    description: "Docker registry url"
     required: false
-    default: 'registry.vuestorefront.cloud'
+    default: "registry.storefrontcloud.io"
+  docker_registry_ref:
```

Comment:
Can't we by default compute the value? It should be like:
`${{ inputs.project_name }}-storefrontcloud-io`

Reply:
it won't build if i use this change unfortunately. i can't seem to reference inputs within the inputs section of this file. it has to be hard coded. 

However within the codebase the continuous-delivery.yml file looks like this

`docker_registry_ref: ${{ env.DOCKER_REGISTRY_URL }}/${{ env.CUSTOMER_NAME }}-storefrontcloud-io`